config/routes.rb

```diff
@@ -1,10 +1,7 @@
-require 'staff_constraint'
-
 require 'sidekiq/web'
 
 Rails.application.routes.draw do
-  mount Peek::Railtie => '/peek',    constraints: StaffConstraint.new unless Rails.env.test?
-  mount Sidekiq::Web  => '/sidekiq', constraints: StaffConstraint.new
+  mount Peek::Railtie => '/peek'
```

Comment:
With the routing constraint removed, is this endpoint still being authenticated?